openwisp_notifications/templates/openwisp_notifications/emails/notification.html

```diff
@@ -0,0 +1,149 @@
+{% extends "openwisp_utils/email_template.html" %}
+{% load static %}
+
+{% block styles %}
+    {{ block.super }}
+    <style type="text/css">
+        .alert {
+            box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.25);
+            border-radius: 8px;
+            border: outset 1px #eee;
+            padding: 1.5rem 1.25rem;
+            font-size: 0.75rem;
+            width: 100%;
+            color: #475467;
+        }
+
+        .alert-link {
+            display: block;
+            text-decoration: none;
+        }
+
+        .alert-link+.alert-link {
+            margin-top: 0.75rem;
+        }
+
+        .alert div+div {
+            margin-top: 0.25rem;
+        }
+
+        .alert .timestamp {
+            font-weight: 400;
+            line-height: 1.25rem;
+            margin: 0px;
+        }
+
+        .alert .badge {
+            background-color: #F2F4F7;
+            border: solid #D0D5DD 1px;
+            color: #333;
+            border-radius: 16px;
+            padding: 0.13rem 0;
+            line-height: 1.13rem;
+            text-transform: capitalize;
+            display: inline-block;
+            text-align: center;
+            min-width: 4rem;
+        }
+
+        .alert .badge.warning {
+            color: #DC6803;
+            border-color: #FEC84B;
+            background-color: #FFFAEB;
+        }
+
+        .alert .badge.warning {
+            color: #DC6803;
+            border-color: #FEC84B;
+            background-color: #FFFAEB;
+        }
+
+        .alert .badge.error {
+            color: #D92D20;
+            border-color: #FDA29B;
+            background-color: #FEE4E2;
+        }
+
+        .alert .badge.success {
+            color: #067647;
+            border-color: #75E0A7;
+            background-color: #DCFAE6;
+        }
+
+        a:link:hover .alert,
+        a:link:focus .alert {
+            background-color: #fff9c466
+        }
+
+        .alert .title {
+            margin-left: 0.5rem;
+            padding: 0.13rem 0;
+        }
+
+        .alert .title p {
+            line-height: 1.25rem;
+            font-size: 0.875rem;
+            display: inline-block;
+            vertical-align: middle;
+            margin: 0 auto;
+            overflow: hidden;
+            text-overflow: ellipsis;
+            color: #0F1729;
+            max-width: 550px;
+            width: 100%;
+            white-space: nowrap;
+        }
+
+        .alert .title {
+            display: inline-block;
+            width: 80%;
+        }
+
+        table.alert tr td:last-child {
+            width: 6%;
+        }
+
```

Comment:
can we avoid these blank lines to reduce email size? Aren't we using indentation of 2 spaces in HTML templates?